### PR TITLE
chore: add an exmaple: clone with ssh

### PR DESCRIPTION
**Pull request by fatelei**
RT

### CODE REVIEW

**Comment by fatelei** [comment]
@mcuadros  Add an example for git clone with ssh. Failed test because net dial.

File: _examples/clone/auth/ssh/main.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		password = os.Args[4]
+	}
+
+	_, err := os.Stat(privateKeyFile)
+	if err != nil {
+		Warning("read file %s failed %s\n", privateKeyFile, err.Error())
+		return
+	}
+
+	// Clone the given repository to the given directory
```
Comment (mcuadros) [inline _examples/clone/auth/ssh/main.go:26]: this should be the equivalent git command, not a comment

File: _examples/clone/auth/ssh/main.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	Info("git clone %s %s using ssh", url, directory)
```
Comment (mcuadros) [inline _examples/clone/auth/ssh/main.go:27]: this should be the equivalent git command, not a comment

Comment (fatelei) [reply _examples/clone/auth/ssh/main.go:27]: ok

Comment (fatelei) [reply _examples/clone/auth/ssh/main.go]: resolved

**Pull request merged**